calibrators/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,157 @@
+    <td><img src="../images/tag_based_pnp_calibrator/calibrated_tf.jpg" alt="calibrated_tf" width = 500px ></td>
+   </tr>
+   <tr>
+    <td><p style="text-align: center;">Before Calibration.</p></td>
+    <td><p style="text-align: center;">After Calibration.</p></td>
+  </tr>
+</table>
+
+## FAQ
+
+- Why doesn't the calibrator add calibration pairs?
+
+  - One possible reason is that the current pair is too close to previously collected data. In that case, the current data is not accepted.
+  - The timestamps of the lidar and camera are not synchronized, this can be checked with `ros2 topic echo [topic_name] --field header.stamp`. Setting the parameter `use_receive_time` to `True` might help to solve the issue but is not recommended as a long-term solution.
+  - The detections are not stable enough. This can happen due to the following reasons:
+    - The detection rate is not stable. The lidar points inside the lidar frames are not sufficient for the algorithm to detect the tags reliably, which usually happens when the tag is far away from the sensor or outside is high-density zone. Please move the tag to a position where enough points (scan lines) hit the tag. If the user forcefully calibrates under these conditions, the results can be compromised.
+    - The tag is physically unstable. Due to wind, mount, or other external, even if the detector works correctly, the detection will not converge. If this is the case, please remove these external factors before attempting calibration. If the user forcefully calibrates under these conditions, the results can be compromised.
+
+- Why doesn't the UI launch?
+
+  - Check with `ros2 node list` if the relevant nodes have been started. It is possible that the provided parameters don’t match any of the valid arguments.
+  - If the UI crashes (check the console for details), it is probably due to a bad PySide installation, invalid intrinsic parameters, invalid extrinsic parameters, etc.
+  - The timestamps of the lidar and camera are not synchronized.
+
+- Why does the reprojection error increase when more data is collected?
+
+  - When there are few samples, the model will fit the available data the best it can, even in the presence of noise (over-fitting). The more data is collected, the error may increase to a certain extent, but that corresponds to the model attempting to fit all the data, this time unable to fit the noise, resulting in a higher error. However, it should reach a more-or-less table peak with about 10-15 pairs (depending on the data collection pattern/sampling).
+
+- Why doesn't the reprojection error seem low enough?
+
+  - The intrinsics may not be accurate, thus limiting the performance of the method.
+  - The boards are not appropriate (are bent).
+  - The boards moved too much while calibrating.
+  - The lidar detections were not very good. Try collecting detections in areas where there is more resolution.
+  - Check the rviz whether there are some outliers.
```

Comment:
This last point is not connected well (does not answer the question you placed)

Reply:
I removed it in https://github.com/tier4/CalibrationTools/pull/164/commits/f913db4f257b63b2686f2409c4ead6b1dbc1f4bc. I am not sure whether outliers show before.

Reply:
The item (outliers) is correct, just not the way it was phrased
We have had outliers when we force the system to detect far-away targets or use prototype lidars

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/bd4a44004508d978cfad397ffc74f4810389212c